specification/resources/resource-manager/Microsoft.Resources/stable/2018-06-01/subscriptions.json

```diff
@@ -0,0 +1,412 @@
+      },
+      "description": "Location information."
+    },
+    "LocationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Location"
+          },
+          "description": "An array of locations."
+        }
+      },
+      "description": "Location list operation response."
+    },
+    "Subscription": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID for the subscription. For example, /subscriptions/00000000-0000-0000-0000-000000000000."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription display name."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription tenant ID."
+        },
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription state. Possible values are Enabled, Warned, PastDue, Disabled, and Deleted.",
+          "enum": [
+            "Enabled",
+            "Warned",
+            "PastDue",
+            "Disabled",
+            "Deleted"
+          ],
+          "x-ms-enum": {
+            "name": "SubscriptionState",
+            "modelAsString": false
+          }
+        },
+        "subscriptionPolicies": {
+          "$ref": "#/definitions/SubscriptionPolicies",
+          "description": "The subscription policies."
+        },
+        "authorizationSource": {
+          "type": "string",
+          "description": "The authorization source of the request. Valid values are one or more combinations of Legacy, RoleBased, Bypassed, Direct and Management. For example, 'Legacy, RoleBased'."
+        }
+      },
+      "description": "Subscription information."
+    },
+    "SubscriptionPolicies": {
+      "properties": {
+        "locationPlacementId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription location placement ID. The ID indicates which regions are visible for a subscription. For example, a subscription with a location placement Id of Public_2014-09-01 has access to Azure public regions."
+        },
+        "quotaId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription quota ID."
+        },
+        "spendingLimit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription spending limit.",
+          "enum": [
+            "On",
+            "Off",
+            "CurrentPeriodOff"
+          ],
+          "x-ms-enum": {
+            "name": "spendingLimit",
+            "modelAsString": false
```

Comment:
see my previous comment on x-ms-enum

Reply:
This would also break compatibility once new SDKs get generated.